clustering_key_filter.hh

```diff
@@ -63,6 +63,10 @@ public:
         }
         return clustering_key_filter_ranges(ranges);
     }
+    static clustering_key_filter_ranges get_maybe_reversed_ranges(const schema& schema, const query::partition_slice& slice, const partition_key& key) {
```

Comment:
Why is this method called `maybe`? It never reverses, does it?

Reply:
It does not reverse the order of ranges, but in case we have `reversed` in the `partition_slice`, the order of ranges has been actually reversed beforehand, so if we do not reverse them again, we actually return them in reversed order